src/headers.js

```diff
@@ -130,47 +131,20 @@ class Headers {
   }
 
   forEach (callback, thisArg) {
-    this[kHeaders] = sort(this[kHeaders])
+    this[kHeaders] = sort1d(this[kHeaders])
```

Comment:
```suggestion
    sort1d(this[kHeaders])
```
remove the reassignment since the algorithm mutates in place